applications/FluidDynamicsApplication/custom_elements/symbolic_stokes.cpp

```diff
@@ -198,6 +198,85 @@ void SymbolicStokes<TElementData>::AddBoundaryTraction(
     noalias(rRHS) += prod(rData.lhs,values);
 }
 
+template <class TElementData>
+void SymbolicStokes<TElementData>::Calculate(
+    const Variable<double> &rVariable,
+    double &rOutput,
+    const ProcessInfo &rCurrentProcessInfo)
+{
+    KRATOS_TRY
+    rOutput = 0.0;
+
+    if (rVariable == HEAT_FLUX) {
+        TElementData data;
+        data.Initialize(*this, rCurrentProcessInfo);
+        rOutput = 0.0;
+        // Shape functions
+        const GeometryType &r_geometry = this->GetGeometry();
+        const unsigned int num_nodes = r_geometry.PointsNumber();
+        Vector data_N(num_nodes);
+        for (unsigned int i = 0; i < num_nodes; i++) {
+            data_N[i] = 1.0 / (double)num_nodes;
+        }
+        data.N = data_N;
+        // Shape functions gradients
+        const GeometryData::IntegrationMethod integration_method = GeometryData::GI_GAUSS_2;
+        ShapeFunctionDerivativesArrayType shape_derivatives;
+        Vector DetJ;
+        r_geometry.ShapeFunctionsIntegrationPointsGradients(shape_derivatives, DetJ, integration_method);
+        data.DN_DX = shape_derivatives[0]; // Note: Valid only for linear elements

```

Comment:
`GeometryUtils` should be used in here once #8772 is done.

Reply:
I could not use `CalculateGeometryData` from `GeometryUtils` as the geometry is expected to be tetrahedra.  I need it for prisms also.

Reply:
Agree. I didn't remember that the Stokes element was specialized for wedges too. Forget about my comment.